const handleError = (source, error) => {
  console.log(`An error occurd in ${source}: `, error);
};

module.exports = {
  handleError,
};
